templates/footer.html

```diff
@@ -55,14 +55,14 @@ <h4>An Initiative by</h4>
 {#        </div>#}
       </div>
       <div class="social-links text-center text-md-right pt-3 pt-md-0">
-        <a href="https://twitter.com/vkssfalumni" class="twitter" target="_blank"><i class="bx bxl-twitter"></i></a>
+        <a href="https://twitter.com/Sushiksha3" class="twitter" target="_blank"><i class="bx bxl-twitter"></i></a>
         <a href="https://www.facebook.com/vkssfalumni/" class="facebook" target="_blank"><i class="bx bxl-facebook"></i></a>
         <a href="https://www.instagram.com/vkssf_alumni/" class="instagram" target="_blank"><i class="bx bxl-instagram"></i></a>
-        <a href="mailto:sushikshawkc@gmail.com" class="google-plus" target="_blank"><i class="bx bxl-skype"></i></a>
+        <a href="mailto:sushikshawkc@gmail.com" class="google-plus" target="_blank"><i class="bx bxl-email"></i></a>
```

Comment:
bxl-email should be changed to bxl-mail-send
![image](https://user-images.githubusercontent.com/64249185/99477701-0bc25980-2979-11eb-8ac0-20d8aa90c88d.png)